pkg/identity/gitlabcom/principal_test.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2023 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gitlabcom
```

Comment:
Any reason for not just having `gitlab` as the package name without the TLD (like github and buildkite)?

Reply:
Maybe to make it clear this is scoped to gitlab.com (the hosted instance) for now and custom gitlab deployments are not supported (yet)?

Reply:
we can rename that, did this in the beginning because i don't know how we want to deal with GitLab self-hosted instances

Reply:
All the tokens should have the same shape, just like Kubernetes tokens, but the issuer will vary.